BankABC.cpp

```diff
@@ -204,15 +204,24 @@ inline void Transaction::setAmount(double amountTr)
 //****************************************************************************
 void sortAccounts(BankAccount ** list)
 {
-
-
-
-
-
-
-
-
-
+	int i = 0;
+	for (i = 0; i <= K_SizeMax - 1; i++) {
+		for (int j = i + 1; j <= K_SizeMax; j++) {
+			if (*(list + j) == NULL) {
+				break;
+			}
+			int accId1 = (*(list + i))->getAccountId();
+			int accId2 = (*(list + j))->getAccountId();
+			accId1 == 0 ? accId1 = 999999;
```

Comment:
whats happening here?

Reply:
yea kevin wut

Reply:
0 needs to be at the end so i check if the accountID is 0 then "make" it look like the largest number

Reply:
noob

Reply:
dont you need a `:` with that operator?

Reply:
Why. You don't need a else in this case since the variable already the value.

Reply:
the variable already the value

Reply:
I like how you never have anything useful to contribute :)

Reply:
cuz ur code has nothing useful to contribute

Reply:
what's wrong with his code?

Reply:
It was giving me errors so I thought maybe its broken